lib/churn/calculator.rb

```diff
@@ -271,7 +271,15 @@ def reject_low_churn_files(files)
     end
 
     def reject_ignored_files(files)
-      files.reject{ |file, _| @ignores.any?{ |ignore| /#{ignore}/ =~ file } }
+      files.reject do |file, _|
+         @ignores.any? do |ignore|
+           begin
+             /#{ignore}/ =~ file
+           rescue RegexpError
```

Comment:
@danmayer One idea for this. Maybe you want to report something to the console? That way the user can know that their ignore regex is not being used. What do you think?

Reply:
yeah makes sense I will add this tomorrow and then do a release